app/controllers/payment_transactions_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PaymentTransactionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @payment = Payment.find(params[:payment_id])
+    @transaction = PaymentTransaction.new
+  end
+
+  def edit
+    @payment = Payment.find(params[:payment_id])
+    @transaction = PaymentTransaction.find_by(payment_id: [params[:payment_id]])

```

Comment:
Será que não daria pra fazer algo como: `@transaction = @payment.payment_transaction`. Ficaria mais legal que bater no banco duas vezes. O que acha? Vale pra outro uso abaixo também.

Fiquei na dúvida se um pagamento pode ter só uma ou muitas transações porque se for mais de uma esse find_by pode falhar também.